templating/twig_extension_repository.rst

```diff
@@ -0,0 +1,93 @@
+.. index::
+   single: Using the Twig Extensions Repository
```

Comment:
I wonder if we really need this article. We don't explain those extensions (because they are documented in their own repo) and they are not special regarding the Symfony+Twig config: they are enabled just like any other Twig extensions.